### DISCUSSION TITLE
Welcome to kymatio Discussions!

### DISCUSSION THREAD

**Discussion by edouardoyallon** [kymatio/kymatio#684] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lostanlen** [comment]
Hi everyone, i'm Vincent @Lostanlen and i am a CNRS scientist at LS2N, the Laboratory of Digital Sciences of Nantes (France). My main research focus is the analysis of 1D signals: primarily, music, animal vocalizations, and electroencephalography.

I did my PhD at ENS between 2013 and 2017, around the same time as @edouardoyallon :)
During that time, i worked with @janden and Stéphane Mallat on developing the joint time--frequency scattering transform (jTFS), with applications to supervised classification and audio texture synthesis. I have also worked with Mathieu Lagrange and Changhong Wang on applying scattering transforms to the analysis of instrumental playing techniques.

I am the computer music assistant of a German avant-garde composer named Florian Hecker. Him and i have produced computer-generated music by gradient backpropagation in a deep scattering network. One example is "The Shape of RemiXXXes to Come", a vinyl which came out on Warp Records in 2018: https://lorenzosenni.bandcamp.com/album/the-shape-of-remixxxes-to-come

Thus far, i have used it for two scientific applications:
* wav2shape, a neural-physical drum synthesizer, made by Han Han and myself
* a study of auditory masking named "One or Two Components?", made by Alice Cohen-Hadria, Juan Pablo Bello, and myself

You can learn more about my research at: https://www.lostanlen.com

In the future, i would like to integrate jTFS as part of Kymatio. I would also like to help with the documentation and the gallery of examples of Kymatio in order to make the library more accessible to newcomers. On a related note, i would like to co-organize a Kymatio workshop in France around spring/summer 2022. If you wish to volunteer as a co-organizer, please reach out directly to me.

**Comment by Emvlt** [comment]
Hey everyone! My name is Emilien Valat, I'm a PhD student in engineering at University of Southampton:)

I have just discovered Kymatio and began to use it for processing 1D signals as part of a project to reduce X-Ray intake in Computed Tomography using DL techniques. The job done is awesome and easy to use, well done!
Being completely new to scattering networks and wavelet transform, I was wondering if anyone could give me resources/tips&tricks on how to:

- design a loss function in the "scattered" space (if that makes sense ahaha)
- choose the values of Q and J of the Scattering1D function

Not sure how i can contribute to the project but willing to help out if anyone has a suggestion!
Any insight is greatly appreciated:)

Thanks in advance,

Emilien
